api/cinemaHall/cinemaHallRoute.js

```diff
@@ -109,6 +118,13 @@ router.delete(
             .json({ msg: 'Can not delete seats' });
         }
       });
+      CinemaHall.deleteScreenings(req.params.id, (err) => {
+        if (err) {
+          return res
+            .status(400)
+            .json({ msg: 'Can not delete screenings' });
+        }
+      });
```

Comment:
I like that it's so simple! We should probably delete the orders and the tickets connected with the screening also.